### ISSUE TITLE
gb silently fails on build errors

### ISSUE THREAD

**Issue by kelseyhightower** [constabulary/gb#131]
```
$ gb build -v -a
DEBUG project root "/Users/kelseyhightower/confd"
DEBUG matchPackages: [/Users/kelseyhightower/confd/src] ...
DEBUG args: [github.com/kelseyhightower/confd github.com/kelseyhightower/confd/backends github.com/kelseyhightower/confd/backends/consul github.com/kelseyhightower/confd/backends/dynamodb github.com/kelseyhightower/confd/backends/env github.com/kelseyhightower/confd/backends/etcd github.com/kelseyhightower/confd/backends/redis github.com/kelseyhightower/confd/backends/zookeeper github.com/kelseyhightower/confd/integration/zookeeper github.com/kelseyhightower/confd/log github.com/kelseyhightower/confd/resource/template]
DEBUG skipping "github.com/kelseyhightower/confd" because of error "no buildable Go source files in /Users/kelseyhightower/confd/vendor/src/github.com/BurntSushi/toml"
DEBUG loadPackage: github.com/hashicorp/consul/api false (/Users/kelseyhightower/confd/vendor/src/github.com/hashicorp/consul/api)
DEBUG loadPackage: github.com/kelseyhightower/confd/backends/consul false (/Users/kelseyhightower/confd/src/github.com/kelseyhightower/confd/backends/consul)
DEBUG loadPackage: github.com/awslabs/aws-sdk-go/aws/awserr true (/Users/kelseyhightower/confd/vendor/src/github.com/awslabs/aws-sdk-go/aws/awserr)
DEBUG loadPackage: github.com/awslabs/aws-sdk-go/internal/apierr true (/Users/kelseyhightower/confd/vendor/src/github.com/awslabs/aws-sdk-go/internal/apierr)
DEBUG skipping "github.com/kelseyhightower/confd/backends" because of error "no buildable Go source files in /Users/kelseyhightower/confd/vendor/src/github.com/vaughan0/go-ini"
DEBUG skipping "github.com/kelseyhightower/confd/backends/dynamodb" because of error "no buildable Go source files in /Users/kelseyhightower/confd/vendor/src/github.com/vaughan0/go-ini"
DEBUG loadPackage: github.com/kelseyhightower/confd/backends/env false (/Users/kelseyhightower/confd/src/github.com/kelseyhightower/confd/backends/env)
DEBUG loadPackage: github.com/coreos/go-etcd/etcd false (/Users/kelseyhightower/confd/vendor/src/github.com/coreos/go-etcd/etcd)
DEBUG loadPackage: github.com/kelseyhightower/confd/backends/etcd false (/Users/kelseyhightower/confd/src/github.com/kelseyhightower/confd/backends/etcd)
DEBUG loadPackage: github.com/garyburd/redigo/internal false (/Users/kelseyhightower/confd/vendor/src/github.com/garyburd/redigo/internal)
DEBUG loadPackage: github.com/garyburd/redigo/redis false (/Users/kelseyhightower/confd/vendor/src/github.com/garyburd/redigo/redis)
DEBUG loadPackage: github.com/kelseyhightower/confd/backends/redis false (/Users/kelseyhightower/confd/src/github.com/kelseyhightower/confd/backends/redis)
DEBUG loadPackage: github.com/samuel/go-zookeeper/zk false (/Users/kelseyhightower/confd/vendor/src/github.com/samuel/go-zookeeper/zk)
DEBUG loadPackage: github.com/kelseyhightower/confd/backends/zookeeper false (/Users/kelseyhightower/confd/src/github.com/kelseyhightower/confd/backends/zookeeper)
DEBUG loadPackage: github.com/kelseyhightower/confd/integration/zookeeper true (/Users/kelseyhightower/confd/src/github.com/kelseyhightower/confd/integration/zookeeper)
DEBUG skipping "github.com/kelseyhightower/confd/log" because of error "no buildable Go source files in /Users/kelseyhightower/confd/vendor/src/github.com/Sirupsen/logrus"
DEBUG skipping "github.com/kelseyhightower/confd/resource/template" because of error "no buildable Go source files in /Users/kelseyhightower/confd/vendor/src/github.com/BurntSushi/toml"
DEBUG buildPackage: github.com/kelseyhightower/confd/backends/consul
DEBUG buildPackage: github.com/hashicorp/consul/api
DEBUG buildPackage: github.com/kelseyhightower/confd/backends/env
DEBUG buildPackage: github.com/kelseyhightower/confd/backends/etcd
DEBUG buildPackage: github.com/coreos/go-etcd/etcd
DEBUG buildPackage: github.com/kelseyhightower/confd/backends/redis
DEBUG buildPackage: github.com/garyburd/redigo/redis
DEBUG buildPackage: github.com/garyburd/redigo/internal
DEBUG buildPackage: github.com/kelseyhightower/confd/backends/zookeeper
DEBUG buildPackage: github.com/samuel/go-zookeeper/zk
DEBUG buildPackage: github.com/kelseyhightower/confd/integration/zookeeper
DEBUG {main github.com/kelseyhightower/confd/integration/zookeeper /Users/kelseyhightower/confd/src/github.com/kelseyhightower/confd/integration/zookeeper} is a main package, not installing
INFO github.com/kelseyhightower/confd/integration/zookeeper
DEBUG gc:gc github.com/kelseyhightower/confd/integration/zookeeper /Users/kelseyhightower/confd /var/folders/ds/jbxmdvmx0rj1z7vc4br6j97h0000gn/T/gb915817495/github.com/kelseyhightower/confd/integration/zookeeper/main.a [src/github.com/kelseyhightower/confd/integration/zookeeper/main.go]
DEBUG cd /Users/kelseyhightower/confd; [/usr/local/go/pkg/tool/darwin_amd64/6g -p github.com/kelseyhightower/confd/integration/zookeeper -pack -o /var/folders/ds/jbxmdvmx0rj1z7vc4br6j97h0000gn/T/gb915817495/github.com/kelseyhightower/confd/integration/zookeeper/main.a -I /var/folders/ds/jbxmdvmx0rj1z7vc4br6j97h0000gn/T/gb915817495 -I /Users/kelseyhightower/confd/pkg/darwin/amd64 -complete src/github.com/kelseyhightower/confd/integration/zookeeper/main.go]
DEBUG cd .; [/usr/local/go/pkg/tool/darwin_amd64/6l -o /Users/kelseyhightower/confd/bin/zookeeper -L /var/folders/ds/jbxmdvmx0rj1z7vc4br6j97h0000gn/T/gb915817495 -L /Users/kelseyhightower/confd/pkg/darwin/amd64 -extld=gcc /var/folders/ds/jbxmdvmx0rj1z7vc4br6j97h0000gn/T/gb915817495/github.com/kelseyhightower/confd/integration/zookeeper/main.a]
DEBUG removing work directory: /var/folders/ds/jbxmdvmx0rj1z7vc4br6j97h0000gn/T/gb915817495
DEBUG build duration: 327.099683ms map[compile:25.094244ms link:285.695742ms]
confd $ echo $?
0
```

**Comment by davecheney** [comment]
@kelseyhightower where is the build error ?

**Comment by kelseyhightower** [comment]
```
DEBUG skipping "github.com/kelseyhightower/confd/resource/template" because of error "no buildable Go source files in /Users/kelseyhightower/confd/vendor/src/github.com/BurntSushi/toml"
```

**Comment by davecheney** [comment]
Thanks.

**Comment by pires** [comment]
+1

**Comment by davecheney** [comment]
@kelseyhightower I think the original problem was fixed in #126, when I fixed copypath, but I can replicate the problem.
